sacc/covariance.py

```diff
@@ -192,6 +194,17 @@ def inverted(self):
         """
         return invert_spd_matrix(self.covmat)
 
+    def get_dense(self):
```

Comment:
I've been wanting this method! Do we want to make it a property, like `cov.dense` or is a method better here?

Reply:
Yeah, that's a good idea, I'll make it a property.